account.go

```diff
@@ -186,6 +186,11 @@ type TOSAcceptanceParams struct {
 	UserAgent string `json:"user_agent"`
 }
 
+// RejectionParams is the structure for the Reject function.
+type AccountRejectionParams struct {
```

Comment:
Another nit, but what do you think about renaming this to `AccountRejectParams`? I think it may be more consistent with the way other params are named (e.g. we have `AccountListParams` instead of `AccountListingParams`)